zulipterminal/ui_tools/boxes.py

```diff
@@ -1609,6 +1609,10 @@ def __init__(self, panel_view: Any, search_command: str,
         self.search_text = (
             f"Search [{', '.join(keys_for_command(search_command))}]: "
         )
+        self.search_error = urwid.AttrMap(
+            urwid.Text([' ', STREAM_MARKER_INVALID, ' Not Found']),
```

Comment:
Or maybe we could change its name to INVALID_OPERATION so that it can be used everywhere consistently?

Reply:
@neiljp Should I go with X or a name change as suggested above?

Reply:
I wasn't necessarily meaning to change the symbol, just the text. "Not Found" suggests to me that there is one/none outcome only, but there are many potential matches here.

It would be fine to add a new symbol name, even with the same codepoint, to make this work for all the search boxes and have a more appropriate name.

Reply:
Oh I must have skipped "No results" without noticing it. Yes No result seems more apt here, I'll change the text :+1: